simulation-ros/src/turtlebot2i/turtlebot2i_safety/developing_scripts/parse_sg.py

```diff
@@ -0,0 +1,196 @@
+#!/usr/bin/env python
+#from graphviz import Digraph
+import pydot # sudo pip install -I pydot==1.2.4
```

Comment:
@EnyuCao could you please create a requirements.txt file, please?

see https://docs.python-guide.org/dev/virtualenvs/#lower-level-virtualenv

Reply:
Yes, I will try to do that. I wrote this note it here, because we need this specific version of pydot, not the default pydot. @albertohata, Please help me check the requirements.txt after I creating it.

Reply:
Hi @berezovskyi  , I created this following requirements.txt. But there are a lot of package, so I don't know how many are necessary. Please have a look when you have time. https://ericsson-my.sharepoint.com/personal/enyu_cao_ericsson_com/Documents/requirements.txt

Reply:
It is weird. I have successfully activate the virtual environment and the name appear on the left of the prompt. But even I install nothing with Pip, I can get a long list from pip ($pip list). I set up the virtualenv with this command $ virtualenv --no-site-packages venv, so it should be clean. @albertohata Can you help me on this problem?

Reply:
Here is the new requirements.txt.
It is very simple (two lines): 
pydot==1.2.4
pyparsing==2.2.0
Link:
https://ericsson-my.sharepoint.com/personal/enyu_cao_ericsson_com/Documents/requirements.txt